documentation/markup.rst

```diff
@@ -10,6 +10,30 @@ This document describes the custom reStructuredText markup introduced by Sphinx
 to support Python documentation and how it should be used.
 
 
+Quick Reference
+===============
+
+This table summarizes which markup should be used for some commonly used
+elements:
+
+===================== =========================================== ====================
+Element               Markup                                      See also
+===================== =========================================== ====================
+arguments/parameters  ``*arg*``                                   :ref:`inline-markup`
+vars/literals/code    ````var````, ````42````, ````len(s) - 1```` :ref:`inline-markup`
+True/False/None       ````True````, ````False````, ````None````   :ref:`inline-markup`
+functions             ``:func:`sorted```                          :ref:`roles`
+func definitions      ``.. function:: func(args)``                :ref:`directives`
+reference labels      ``.. _label-name:``                         :ref:`doc-ref-role`
+internal ref          ``:ref:`label-name```                       :ref:`doc-ref-role`
+external links        ```Link text <https://target>`_``           :ref:`hyperlinks`
+roles w/ custom text  ``:role:`custom text <target>```            :ref:`roles`
+roles w/o link        ``:role:`!target```                         :ref:`roles`
+roles w/o first part  ``:role:`~hidden.hidden.visible```          :ref:`roles`
```

Comment:
```suggestion
roles w/only last part  ``:role:`~hidden.hidden.visible```          :ref:`roles`
```

Be more specific/correct, since you'll have the space here if you make the above change